index.html

```diff
@@ -14,54 +14,63 @@
+          <li><a href="#about-section">About</a></li>
+          <li><a href="#contact-section">Contact</a></li>
+        </ul>
+      </nav>
+      <section class="home-page">
+        <h2 class="intro">
+          Hey there. I'm Selma <br /><span
+            >I am a software 
+            developer</span>
+        </h2>
+        <p class="info">
+          I can help you build a product , feature or website Look through some of
+          my work and experience! If you like what you see and have a project you
+          need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="icon-list">
+          <li>
+            <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a>
+          </li>
+          <li>
+            <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i> </a>
+          </li>
+          <li>
+            <a href="#"><i class="fa fa-medium" aria-hidden="true"></i> </a>
+          </li>
+          <li>
+            <a href="#"><i class="fa fa-github" aria-hidden="true"></i> </a>
+          </li>
+          <li>
+            <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i> </a>
+          </li>
+        </ul>
+        <br />
+        <br />
+        <br />
+        <i id="down-arrow" class="linkhover">
+          <img src="images/Disabled.png" alt="down arrow"/>
+        </i>
+      </section>
     </header>
-    <section class="home-page">
-      <h2 class="intro">
-        Hey there. I'm Selma <br /><span
-          >I am a software<br />
-          developer</span
-        >
-      </h2>
-      <p class="info">
-        I can help you build a product , feature or website Look through some of
-        my work and experience! If you like what you see and have a project you
-        need coded, don’t hestiate to contact me.
-      </p>
-      <ul class="icon-list">
-        <li>
-          <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a>
-        </li>
-        <li>
-          <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i> </a>
-        </li>
-        <li>
-          <a href="#"><i class="fa fa-medium" aria-hidden="true"></i> </a>
-        </li>
-        <li>
-          <a href="#"><i class="fa fa-github" aria-hidden="true"></i> </a>
-        </li>
-        <li>
-          <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i> </a>
-        </li>
-      </ul>
-      <br />
-      <br />
-      <br />
-      <i id="down-arrow" class="linkhover"
-        ><img src="images/Disabled.png" alt="down arrow"
-      /></i>
-    </section>
+
     <main>
       <div id="main-head">
         <h2>
-          My Recent <br />
-          Works
+          My Recent Works
         </h2>
-        <hr />
+        <div class="green-line">
+          <img src="images/Indicator.png" alt="green line">
+      </div>
       </div>
+      <div class="work-container">
       <div class="work-div">
         <i class="projectimg"></i>
```

Comment:
- [ ] [OPTIONAL]  Please notice that the languages in cards are wrapping:

Your card  | Original card
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/144694566-f541eebf-bd39-406c-97be-27d6195ea9f4.png)  | ![image](https://user-images.githubusercontent.com/77038610/144694576-07dd7d42-c72f-4cda-9dfb-aca002eb3e92.png)

I suggest to give them more space so that they did not wrap in desktop screens 👍🏻